src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let discount = 0;
```

Comment:
we usually name constants in uppercase
<img width="860" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/80b1d9cd-31fd-41f6-a42a-b5b5b992c78e">